### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by DmitriyMV**
- github.com/aws/aws-sdk-go to v1.44.209
- github.com/stretchr/testify to v1.8.2
- github.com/jsimonetti/rtnetlink to v1.3.1
- github.com/opencontainers/runtime-spec to v1.1.0-rc.1.0.20230215090456-58ec43f9fc39
- google.golang.org/genproto to v0.0.0-20230223222841-637eb2293923
- github.com/emicklei/dot to v1.3.1
- github.com/gdamore/tcell/v2 to v2.6.0
- github.com/insomniacslk/dhcp to v0.0.0-20230220063916-5369909a5de7
- github.com/jsimonetti/rtnetlink to v1.3.1
- github.com/opencontainers/runtime-spec to v1.1.0-rc.1.0.20230215090456-58ec43f9fc39
- github.com/rivo/tview to v0.0.0-20230226195229-47e7db7885b4

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -76,13 +76,13 @@ require (
 	github.com/mdlayher/netlink v1.7.1
 	github.com/mdlayher/netx v0.0.0-20220422152302-c711c2f8512f
 	github.com/opencontainers/image-spec v1.1.0-rc2
-	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
+	github.com/opencontainers/runtime-spec v1.1.0-rc.1.0.20230215090456-58ec43f9fc39
```
Comment (frezbo) [inline go.mod:79]: bumping this last time broke the build, is it fixed now?

Comment (smira) [reply go.mod:79]: I think we should keep containerd's version

**Comment by DmitriyMV** [comment]
> bumping this last time broke the build, is it fixed now?

@smira @frezbo This is now fixed, but it introduces some changes to `Mount` type (backward compatible).

File: go.mod (hunk 1 above)
Comment (DmitriyMV) [reply go.mod]: Reverted.

**Review by smira — APPROVED** [review]

**Pull request merged**